packages/config/config/devices/0x0344/he-ht01.json

```diff
@@ -0,0 +1,795 @@
+					"value": 8
+				},
+				{
+					"label": "Or (920.9 MHz, 921.7 MHz, 923.1 MHz)",
+					"value": 9
+				}
+			]
+		},
+		"3": {
+			"label": "Hardware and Software Versions",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 0,
+			"readOnly": true
+		},
+		"4": {
+			"label": "Energy Consumption",
+			"unit": "kW",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 0,
+			"readOnly": true
+		},
+		"5": {
+			"label": "Display Brightness Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 0
+		},
+		"6": {
+			"label": "Touch Buttons Sensitivity",
+			"description": "Allowable range: 1 (low sensitivity) to 10 (high sensitivity).",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 6
+		},
+		"7": {
+			"label": "Relay Output Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "NO",
+					"value": 0
+				},
+				{
+					"label": "NC",
+					"value": 1
+				}
+			]
+		},
+		"8": {
+			"label": "External Input Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Ignore external input",
+					"value": 0
+				},
+				{
+					"label": "Toggle Switch",
+					"value": 1
+				},
+				{
+					"label": "Toggle Switch Reverse",
+					"value": 2
+				},
+				{
+					"label": "Push Button (Momentary Switch)",
+					"value": 3
+				}
+			]
+		},
+		"9": {
+			"label": "Operating Mode for External Input Action",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 6,
+			"defaultValue": 6,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "COM",
+					"value": 1
+				},
+				{
+					"label": "TIME",
+					"value": 2
+				},
+				{
+					"label": "DRY",
+					"value": 3
+				},
+				{
+					"label": "ECO",
+					"value": 4
+				},
+				{
+					"label": "VAC",
+					"value": 5
+				},
+				{
+					"label": "MAN",
+					"value": 6
+				}
+			]
+		},
+		"10": {
+			"label": "Floor Sensor Resistance",
+			"unit": "kohm",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 100,
+			"defaultValue": 10
+		},
+		"11": {
+			"label": "Source Sensor",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 3,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Air sensor",
+					"value": 1
+				},
+				{
+					"label": "Air and Floor sensors",
+					"value": 2
+				},
+				{
+					"label": "Floor sensor",
+					"value": 3
+				},
+				{
+					"label": "Floor and Air sensor",
+					"value": 4
+				},
+				{
+					"label": "Time regulator",
+					"value": 5
+				},
+				{
+					"label": "Time regulator and Air sensor",
+					"value": 6
+				},
+				{
+					"label": "Time regulator and Floor sensor",
+					"value": 7
+				}
+			]
+		},
+		"12": {
+			"label": "Air Temp Min",
+			"description": "Allowable range: 1 - 36, x10",
```

Comment:
```suggestion
```
This is unnecessary with the unit and min/max value definitions. Please also change the next parameters.